appservice/src/main/java/nu/yona/server/rest/AppleAppSiteAssociationController.java

```diff
@@ -30,17 +28,17 @@
 	private YonaProperties yonaProperties;
 
 	@Autowired
-	private VelocityEngine velocityEngine;
+	@Qualifier("otherTemplateEngine")
+	private TemplateEngine templateEngine;
 
 	@RequestMapping(value = "", method = RequestMethod.GET)
 	@ResponseBody
 	public ResponseEntity<byte[]> getAppleAppSiteAssociation()
 	{
-		Map<String, Object> templateParameters = new HashMap<String, Object>();
-		templateParameters.put("appleAppId", yonaProperties.getAppleAppId());
-		templateParameters.put("esc", new EscapeTool());
-		return new ResponseEntity<byte[]>(VelocityEngineUtils
-				.mergeTemplateIntoString(velocityEngine, "apple-app-site-association.vm", "UTF-8", templateParameters)
-				.getBytes(StandardCharsets.UTF_8), HttpStatus.OK);
+		Context ctx = new Context();
+		ctx.setVariable("appleAppId", yonaProperties.getAppleAppId());
+
+		return new ResponseEntity<>(
```

Comment:
Then we would to provide the model somehow, right? Let's try that another time and first get over this hurdle.